### ISSUE TITLE
Regularization Techniques для уменьшения галлюцинаций

### ISSUE THREAD

**Issue by kharisovilyas** [kharisovilyas/orbit_ai#14]
Уменьшить frequency ошибок, связанных с галлюцинациями.

Внедрить:
1. Loss-penalty за значения, отсутствующие в запросе
Создать rule-based скрипт:
если модель генерирует значение, которого нет в prompt - добавить штраф в loss

Использовать:
Contrastive Decoding + Negative examples

2. JSON-Repair Loss
Специальный auxiliary loss, который:
- penalizes незакрытые скобки
- penalizes пропуски кавычек
- penalizes “слоты вне схемы”

3. Label Smoothing
smoothing = 0.1
снижает overfitting - улучшает обобщение

Результаты:
-  Создан файл regularization_experiments.md
-  Внедрены 1–2 регуляризационных метода
-  Запущен тестовый fine-tuning
-  Сравнение Regularized vs Non-Regularized
